src/nimble.nim

```diff
@@ -1043,6 +1043,56 @@ proc check(options: Options) =
     display("Failure:", "Validation failed", Error, HighPriority)
     quit(QuitFailure)
 
+
+proc sample(options: Options) =
+  ## ~ copies sample files to current directory - does not creates new subdirectory.
+  ##   (copyDir nimblepath/pkgs/pkg/samples/arg to current directory.)
+  ## ~ finds `arg` in subdirectories. "nimble sample pkgname sub"
+  ## ~ looks for pkg version if requested. "nimble sample pkgname '0.26.2' demo1"
+  ## ~ copies the first match found
+  ## ~ overwrites existing files without question
+
+  let installedPkgs = getInstalledPkgsMin(options.getPkgsDir(), options)
+  var version = toVersionRange(options.action.ver)
+
+  for (pkg,meta) in installedPkgs: # pkg exists?
+    if pkg.name == options.action.package and
+      (withinRange(pkg, version) or pkg.version == "#head"): #"#head" == develop 
+        var (dir, name, ext) = splitFile(pkg.myPath)
+        let examplesDir = dir & DirSep & "samples"
+        if not dirExists(examplesDir):
+          display("Problem:", pkg.name & " has no samples.", 
+              priority = HighPriority)
+          raise NimbleQuit(msg: pkg.name & " has no samples.")
```

Comment:
Please use `raiseNimbleError` instead of using `display` + raising NimbleQuit.

Reply:
done.